FAQ update (#6647)
harness/developer-hub: kb/continuous-delivery/continuous-delivery-faqs.md

#### Can you tell me which steps for a custom stage run directly on the delegate?
Usually all actual task executions occur on delegate. You can add shell script steps to execute the command on delegate.

#### Does harness http step support edgegrid auth?

As of now we do not support edgegrid auth

> #### Is it possible to have  web-hook, trigger to fire when a file-store object is changed?

Reviewer: ```suggestion
#### Is it possible to have a webhook that triggers when a file-store object is changed?
```

> Yes, we can [Doc](https://developer.harness.io/docs/platform/triggers/triggers-reference/#conditions-settings)

Reviewer: ```suggestion
Yes, we can. To learn more go to [Conditions Settings in the Triggers Reference Doc](https://developer.harness.io/docs/platform/triggers/triggers-reference/#conditions-settings)
```

> #### how to analyze cloudformation stack before creating

Reviewer: ```suggestion
#### How do you analyze a cloudformation stack before creating it?
```

> To validate the cloudformation stack, you can install aws cli on delegate and use a shell script to validate the template by running :

Reviewer: ```suggestion
To validate the cloudformation stack, you can install the aws cli on the delegate and use a shell script to validate the template by running :
```

``` 
aws cloudformation validate-template --template-body file://<path-to-your-template-file>
```

> #### what does "buffer already closed for writing" 

Reviewer: ```suggestion
#### What does the error "buffer already closed for writing" mean?
```
